Libraries/LibC/pwd.cpp

```diff
@@ -125,4 +125,38 @@ struct passwd* getpwent()
     strncpy(__pwdb_entry->shell_buffer, e_shell.characters(), PWDB_STR_MAX_LEN);
     return __pwdb_entry;
 }
+
+int putpwent(const struct passwd* p, FILE* stream)
```

Comment:
Would be nice to have a manpage for this too (for good boy points :D), but I don't insist

Reply:
I'll open that in another commit. It might be a good idea for us to have a "documentation week" where we all write as many manpages if we can tbh haha.

Reply:
Great idea :)